### ISSUE TITLE
Could not use diagnostics.sh 

### ISSUE THREAD

**Issue by gengqx** [ApolloAuto/apollo#1774]
here is what I got:
```
Module         Topic                              Period    Max       MinException in thread Thread-5:
                      Traceback (most recent call last):
                                                          File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
                                                    self.run()
                                                                File "/home/tmp/ros/lib/python2.7/dist-packages/rospy/timer.py", line 223, in run
                                                                     self._callback(TimerEvent(last_expected, last_real, current_expected, current_real, last_duration))
          File "modules/tools/diagnostics/diagnostics.py", line 77, in callback_timer
         self.stdscr.addstr(0, 80, "Delay", curses.A_BOLD)
                                                          error: addstr() returned ERR
```

**Comment by Capri2014** [comment]
@gengqx try re-just the screen size

**Comment by ycool** [comment]
The problem can be fixed after adjusting screen size and rerun diagnostics.sh. We didn't find a good way to fix this issue without window resizing yet.  Any help is welcome!

**Comment by zhxt** [comment]
I think I have found the reason, this happened when we attempting to use the outside of the terminal window.
It will boom!!! when the terminal window(which the diagnostic.sh script run) width <= 80.

PR #1845 created.

**Comment by zhxt** [comment]
Another simple solution for this is just add one line:
"export COLUMNS=200"
to 
scripts/diagnostics.sh
before this line:
"python modules/tools/diagnostics/diagnostics.py"

**Comment by ycool** [comment]
@zhxt  Thanks a lot for the fix !

**Comment by zhxt** [comment]
@ycool No problem. Happy to help.  ;-)

**Comment by gengqx** [comment]
@ycool  just set
"export COLUMNS=200" would work,
@zhxt  thanks a lot

**Comment by zhxt** [comment]
@gengqx, actually the number which  > 80 would be ok.
This is the simplest way。

I prefer another fix #1845(merged already), which is more "elegant", python side issue fixed in python side, also that make the resize operation when it's really needed.

Anyway, either of them is okay.
